### PR TITLE
Supress collecting/collected

### PR DESCRIPTION
**Pull request by sabidib**
This attempts to suppress collect messages messages by changing the verbosity on the fly

@angelini @cfournie

### CODE REVIEW

File: pytest_redis.py [hunk 1]
```diff
@@ -91,51 +91,50 @@ def perform_collect_and_run(session):
...
     for arg in redis_list:
-        term.write(os.linesep)
         parts = session._parsearg(arg)
         session._initialparts.append(parts)
         session._initialpaths.add(parts[0])
         arg = "::".join(map(str, parts))
         session.trace("processing argument", arg)
         session.trace.root.indent += 1
         try:
+            #Change the verbosity to suppress collect messages
```
Comment (cfournie) [inline pytest_redis.py]: `# Change`; you should probably get some linting done on this project 😉

**Comment by cfournie** [comment]
What does this look like when run?

**Comment by sabidib** [comment]
It should look like:

```
...
the/test/name PASSED
another/the/test/name PASSED
..
```

File: pytest_redis.py (hunk 1 above)
Comment (sabidib) [inline pytest_redis.py]: Great idea, that will be useful :P

**Comment by cfournie** [comment]
Hax upon hax, but since there's only a visual disturbance if these hacks fail, 👍

**Comment by sabidib** [comment]
=(  It's painful, but unless we mess around with the internals of pytest (specifically [this](https://github.com/pytest-dev/pytest/blob/master/_pytest/terminal.py#L252-L272)) we have to roll with this ...

**Pull request merged**